crates/bevy_log/src/lib.rs

```diff
@@ -96,13 +99,21 @@ pub struct LogPlugin {
     /// Filters out logs that are "less than" the given level.
     /// This can be further filtered using the `filter` setting.
     pub level: Level,
+
+    /// Optionally apply extra transformations to the tracing subscriber.
+    /// For example add [`Layers`](tracing_subscriber::layer::Layer)
+    pub update_subscriber: Option<fn(BoxedSubscriber) -> BoxedSubscriber>,
```

Comment:
Not the biggest fan of this naming, but this work. Not a strict blocker.

Reply:
Maybe `extend_tracing_subscriber_fn`?

Reply:
Maybe `configure`/`configure_callback`

Reply:
The current name makes sense to me.`configure_subscriber` maybe?